protocol/protobuf/chat_message.proto

```diff
@@ -180,6 +180,11 @@ message ChatMessage {
     IDENTITY_VERIFICATION = 13;
     // Only local
     SYSTEM_MESSAGE_PINNED_MESSAGE = 14;
-    SYSTEM_MESSAGE_MUTUAL_STATE_UPDATE = 15;
+    // Only local
+    SYSTEM_MESSAGE_MUTUAL_EVENT_SENT = 15;
```

Comment:
Is it safe to replace `SYSTEM_MESSAGE_MUTUAL_STATE_UPDATE` with `SYSTEM_MESSAGE_MUTUAL_EVENT_SENT`, keeping same value `15`? There won't be conflicts with older versions?

I wonder if it should be a new value, marking the old one as deprecated.

Reply:
I suppose it's safe because it was used by local system messages only. But fresh development builds can see wrong messages in history